trunk/src/app/srs_app_server.cpp

```diff
@@ -1227,11 +1226,26 @@ srs_error_t SrsServer::do_on_tcp_client(ISrsListener* listener, srs_netfd_t& stf
         }
     }
 
+    // For RTC TCP connection, use resource executor to manage the resource.
+    SrsRtcTcpConn* raw_conn = dynamic_cast<SrsRtcTcpConn*>(resource);
+    if (raw_conn) {
```

Comment:
Be careful the `raw_conn (SrsRtcTcpConn resource)` and its `conn (SrsSharedResource wrapper)`, the `raw_conn` will be add to `conn_manager`, the tcp connection resource manager of `SrsServer`, in line 1244, and its wrapper, a SrsSharedResource, will created below in line 1233, this wrapper didn't added to any resource manager, but it will be released by `_srs_rtc_manager` in `SrsExecutorCoroutine`'s destructor. Then the `raw_conn` stored in `conn_manager` will became a wild pointer, the good news is that the `conn_manager` will never remove it, so the process will never crash, but it's still a problem, is it?

So, there are two problem: 
1. the `raw_conn` will be added to `conn_manager`, which will never remove it;
2. the `raw_conn` inside `conn_manager` will became wild pointer once its wapper, the `SrsSharedResource` released by `_srs_rtc_manager` inside `SrsExecutorCoroutine`'s  desctructor.

Reply:
Nop, manager never add raw_conn:

```cpp

    // For RTC TCP connection, use resource executor to manage the resource.
    SrsRtcTcpConn* raw_conn = dynamic_cast<SrsRtcTcpConn*>(resource);
    if (raw_conn) {
        // ......
        return err;
    }
```

Reply:
get it.